account_financial_report/report/templates/vat_report.xml

```diff
@@ -76,7 +76,7 @@
                                 t-raw="tag_or_group['net']"
                                 t-options="{'widget': 'monetary', 'display_currency': res_company.currency_id}"
                             />
-                        </div>
+                            </div>
```

Comment:
**chore**: I think previous formatting was correct. In general, to keep the PR clean, we tend to exclude modifications unrelated to the PR at hand. Formatting fixes can be done in their own commit, if necessary, but in this case they should be excluded.

Reply:
Fixed right now, thank you for pointing this out!